src/web/src/browser/index.ts

```diff
@@ -50,6 +50,7 @@ export const runBrowserContext = ({
         treeUrl: github.getBranchTreeUrl(githubRepository),
         sampleDocuments: github.getSampleOscalDocuments(githubRepository),
         developerExampleUrl: github.getDeveloperExampleUrl(githubRepository),
+        getNewIssueUrl: github.getNewIssueUrl(githubRepository),
```

Comment:
This one should just be an object:

```suggestion
        newIssueUrl: github.getNewIssueUrl(githubRepository),
```

Reply:
You bet, rename is in place now.